added pypi-release rule (#87)
cloudevents/sdk-python: RELEASING.md

> # Releasing CloudEvents SDK for Python

Reviewer: Add a link to this in the PR template and/or the main README?

> To properly release a new cloudevents version, we setup github actions for updating
> repo's [PyPi package.](https://pypi.org/project/cloudevents/)

Reviewer 2: ```suggestion
This repository is configured to automatically publish the corresponding [PyPI
package](https://pypi.org/project/cloudevents/) via GitHub Actions.
```

> This git action assumes contributors have updated the proper modules
> (e.g. [setup.py](setup.py)) to reflect the pypi version changes. It will
> automatically build and deploy onto pypi using
> [this pypi github action](https://github.com/pypa/gh-action-pypi-publish). This
> action gets called on pushes to master (such as version branch's being merged
> into master).

Reviewer 2: ```suggestion
To release a new CloudEvents SDK, contributors should bump the `version` in 
[setup.py](setup.py)) to reflect the new release version. On merge, the action
will automatically build and release to PyPI using
[this PyPI GitHub Action](https://github.com/pypa/gh-action-pypi-publish). This
action gets called on all pushes to master (such as a version branch being merged
into master), but only releases a new version when the version number has changed.
```

> After a major version branch is merged into master, contributors are expected to
> manually create a tag/release for the version they'd just merged into master.

Reviewer 2: ```suggestion
After a version update is merged, maintainers are expected to manually create a
corresponding tag/release.
```

Feel free to view [pypi-release.yml](.github/workflows/pypi-release.yml) for

> more information.

Reviewer: nit: more direct wording

```md
View the GitHub workflow [pypi-release.yml](.github/workflows/pypi-release.yml) for
more information.
```

Author: adjusted
